include/alpaka/alpaka.hpp

```diff
@@ -33,6 +33,9 @@
 #include "alpaka/atomic/AtomicUniformCudaHipBuiltIn.hpp"
 #include "alpaka/atomic/Op.hpp"
 #include "alpaka/atomic/Traits.hpp"
+// grid
+#include "alpaka/grid/GridSyncGpuCudaHip.hpp"
+#include "alpaka/grid/Traits.hpp"
```

Comment:
Does it work if you move these three lines after "extent" ?
Everything else should be in alphabetical order.